src/bin/vhost_user_blk.rs

```diff
@@ -165,7 +165,7 @@ impl VhostUserBlkBackend {
 
 impl VhostUserBackend for VhostUserBlkBackend {
     fn num_queues(&self) -> usize {
-        NUM_QUEUES
+        MAX_NUM_QUEUES
```

Comment:
Here, the fixed value, MAX_NUM_QUEUES, is specified in num_queues() which is required by VhostUserHandler to initialize vrings, thus, vhost-user-blk will always have 128 vrings initialized, no matter how many vrings vhost-user-blk device actually use. Although only the vrings as many as vhost-user-blk device defined have actual data handling, it's sensible to make the queue numbers flexible to be defined for vhost-user-blk backend.

Reply:
This is issue, so i can add the queue_num parameter into backend side. thanks!